docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,118 @@
+# Quick Guide
+
+## What you’ll learn
+
+- Deploy the [Content-cache-k8s charm](https://charmhub.io/content-cache-k8s).
+- Relate to [the Hello-kubecon charm](https://charmhub.io/hello-kubecon).
+- Relate to [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/#what-is-ingress) by using [NGINX Ingress Integrator](https://charmhub.io/nginx-ingress-integrator/).
+
+Through the process, you'll inspect the Kubernetes resources created, verify the workload state and assign the Content-cache-k8s to serve your Hello-kubecon instance.
+
+## Requirements
+
+- Juju 3 installed.
+- Juju controller and model created.
+- NGINX Ingress Controller. If you're using [MicroK8s](https://microk8s.io/), this can be done by running the command `microk8s enable ingress`. For more details, see [Addon: Ingress](https://microk8s.io/docs/addon-ingress).
+
+For more information about how to install Juju, see [Get started with Juju](https://juju.is/docs/olm/get-started-with-juju).
+
+### Deploy the Content-cache-k8s charm
+
+Since Content-cache is meant to serve another charm as its cache, we'll use Hello-kubecon as an example.
+
+Deploy the charms:
+
+```bash
+juju deploy content-cache-k8s
+juju deploy hello-kubecon
+```
+
+To see the pod created by the Content-cache-k8s charm, run `kubectl get pods` on a namespace named for the Juju model you've deployed the Content-cache-k8s charm into. The output is similar to the following:
+
+```bash
+modeloperator-98b8cb7df-m6stx   1/1     Running   0          2m20s
+content-cache-k8s-0             3/3     Running   0          2m
+hello-kubecon-0                 0/2     Running   0          36s
+```
+
+Run [`juju status`](https://juju.is/docs/olm/juju-status) to see the current status of the deployment. In the Unit list, you can see that Content-cache-k8s is blocked:
+
+```bash
+content-cache-k8s/0*  blocked   idle   10.1.97.227         Required config(s) empty: backend, site
+hello-kubecon/0*      active    idle   10.1.97.193   
+```
+
+This means that Content-cache-k8s charm isn't integrated with Hello-kubecon yet.
+
+### Relate to the Hello-kubecon charm
+
+Provide integration between Content-cache-k8s and Hello-kubecon by running the following [`juju relate`](https://juju.is/docs/olm/juju-relate) command:
+
+```bash
+juju relate content-cache-k8s:ingress-proxy hello-kubecon
+```
+
+Run `juju status` to see that the message has changed:
+
+```bash
+content-cache-k8s/0*  active    idle   10.1.97.227         Ready
+hello-kubecon/0*      active    idle   10.1.97.193
+```
+
+Note: `ingress-proxy` is the name of the relation. This is needed because establishes that the two charms are compatible with each other.  You can run `juju info content-cache-k8s` to check what are the relation names provided by the Content-cache-k8s application.
```

Comment:
`This is needed because establishes that the two charms are compatible with each other.`
Is this correct ? Sounds like a strange phrasing

Reply:
Also `juju status --relations` will show relations